documentation/contributing/future_projects/index.markdown

```diff
@@ -204,5 +196,5 @@ For this, we need to identify and implement standard tasks at varying levels of
 - **Prerequisites**: ROS, some motion planning background
 - **Programming skills**: C++
 - **Difficulty**: Medium
-- **Potential mentors**: Nathan Brooks
-- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such a common functionality is not ideal at all. We also extend to cover non-holonomic drive robots. [This is an FAQ](https://github.com/ros-planning/moveit/issues/375) among MoveIt users.
+- **Potential mentors**: Mark Moll, David Lu!!
+- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such a common functionality is not ideal at all. We also extend to cover non-holonomic drive robots. [This is an FAQ](https://github.com/ros-planning/moveit/issues/375) among MoveIt users. Some initial work for this is currently underway for the [Hello Robot Stretch platform](https://hello-robot.com/product), but more work is needed to generalize to other types of mobile bases.
```

Comment:
Is there a link to read up what happened in the initial work? I'm not sure the scope is clear to someone reading this.

```suggestion
- **Description**: The goal is to provide a standard way for integrating mobile base into move group that typically handles arms. While there's a non-standard way to incorporate holonomic drive that requires modifying your robot model ([example](https://groups.google.com/forum/#%21searchin/moveit-users/virtual$20base/moveit-users/G7vG7r23YzI/vpMUoa1hza4J)), customizing your robot package for such a common functionality is not ideal at all. We also extend to cover non-holonomic drive robots. [This is an FAQ](https://github.com/ros-planning/moveit/issues/375) among MoveIt users. Some initial work for this is currently underway for the [Hello Robot Stretch platform](https://hello-robot.com/product), but more work is needed to generalize to other types of mobile bases.

**Related Github issues**: [375](https://github.com/ros-planning/moveit/issues/375)
```

Reply:
It's ongoing work. By the time GSoC starts, we'll definitely have more. PickNik is working with the Hello Robot team on ROS 2 / MoveIt 2 integration and plans to have a demo at the next ROS 2 release in May.